src/federation/query.rs

```diff
@@ -1,144 +0,0 @@
-use anyhow::Context;
-use axum::body::Body;
-use axum::extract::State;
-use axum::http::HeaderMap;
-use axum::response::{IntoResponse, Response};
-use axum::Json;
-use axum_auth::AuthBearer;
-use hex::ToHex;
-use serde::Serialize;
-use sqlx::postgres::any::AnyTypeInfoKind;
-use sqlx::{query, Any, Column, Database, Row};
-use tracing::debug;
-use tracing::log::info;
-
-use crate::federation::observer::FederationObserver;
-use crate::AppState;
-
-pub async fn run_query(
```

Comment:
I removed the SQL query functionality entirely since tools like metabase do a much better job at it.